Packs/CiscoStealthwatch/Playbooks/playbook-block_domain-Cisco_Stealthwatch.yml

```diff
@@ -0,0 +1,232 @@
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: b124cfe7-a56f-4490-880b-78cf3274b99b
+    type: start
+    task:
+      id: b124cfe7-a56f-4490-880b-78cf3274b99b
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      "#none#":
+      - "5"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -100
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 1268fa81-6e6d-4d75-8f22-f6893af3cb9c
+    type: condition
+    task:
+      id: 1268fa81-6e6d-4d75-8f22-f6893af3cb9c
+      version: -1
+      name: Is Cisco Stealthwatch enabled?
+      description: Verify that there is a valid instance of Check Point Firewall enabled.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "#default#":
+      - "4"
+      "yes":
+      - "3"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: modules
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: Stealthwatch Cloud
+                    ignorecase: true
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.state
+                      iscontext: true
+                    right:
+                      value:
+                        simple: Active
+                    ignorecase: true
+                accessor: brand
+            iscontext: true
+          right:
+            value: {}
+          ignorecase: true
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 250
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 951938f0-4b63-46e8-8fdc-7e6a26bbf853
+    type: regular
+    task:
+      id: 951938f0-4b63-46e8-8fdc-7e6a26bbf853
+      version: -1
+      name: Block Domain
+      description: Add a domain or IP to the block list
+      script: Stealthwatch Cloud|||sw-block-domain-or-ip
+      type: regular
+      iscommand: true
+      brand: Stealthwatch Cloud
+    nexttasks:
+      "#none#":
+      - "4"
+    scriptarguments:
+      domain:
+        simple: ${inputs.Domain}
```

Comment:
Please use the UI filters&transformers and pick the input.
![image](https://user-images.githubusercontent.com/28757135/183605445-0ffeae35-5fe4-4384-bbaa-3b1b35e93b84.png)